openassessment/xblock/config_mixin.py

```diff
@@ -121,4 +119,12 @@ def is_mobile_support_enabled(self):
         """
         Returns a boolean indicating if the mobile support feature flag is enabled or not.
         """
-        return self._settings_toggle_enabled(FEATURE_TOGGLES_BY_FLAG_NAME.get(MOBILE_SUPPORT))
+        # .. toggle_name: FEATURES['ENABLE_ORA_MOBILE_SUPPORT']
+        # .. toggle_implementation: SettingToggle
+        # .. toggle_default: False
+        # .. toggle_description: Set to True to enable the ORA2 Xblock to be rendered
+        #     in mobile apps.
+        # .. toggle_use_cases: open_edx
+        # .. toggle_creation_date: 2020-10-14
+        # .. toggle_tickets: https://github.com/edx/edx-ora2/pull/1445
+        return settings.FEATURES.get('ENABLE_ORA_MOBILE_SUPPORT', False)
```

Comment:
Sorry I missed this, but consider updating (or following up) with a refactor to declare a SettingDictToggle at the top of this file, and using it here.  See https://edx.readthedocs.io/projects/edx-toggles/en/latest/decisions/0003-django-setting-toggles.html#decision

Reply:
@morenol: If you want to do the SettingDictToggle refactor, we should probably do it on a separate PR so we can land this.  I'm just waiting for confirmation from the owners on making `ENABLE_ORA_MOBILE_SUPPORT` only work with a Django Setting.

Reply:
I think that it only can be used through a django setting. The previous attempt to make this use a waffle flag failed because the creation of docker images failed.

 This decorator is evaluated on import time: 
https://github.com/edx/edx-ora2/blob/76269007c4d7157ab62a50c48f2715f90e01dca2/openassessment/xblock/openassessmentblock.py#L495